forms/swipe.json

```
@@ -2,30 +2,29 @@
   {
     "name": "_training",
     "context": "Pour répondre",
-    "body": "<b>Oui</b> ✔ ➡<br><br>⬅ <b>Non</b> ❌<br><br>◀ <b>Retour arrière</b>"
+    "body": "<b>Oui</b> ✔ ➡<br><br>⬅ <b>Non</b> ❌<br><br>◀ <b>Corriger la réponse précédente</b>"
   },
   {
     "name": "inquiet",
     "body": "Je suis inquiet",
-    "icon": "😕"
+    "icon": "😕",
+    "positive": "inquietProches",
+    "negative": "sommeil"
   },
   {
     "name": "inquietProches",
     "body": "Inquiet pour la santé de mes proches",
     "icon": "👨‍👩‍👧‍👦",
```

La virgule n'est pas autorisée en fin de d'objet.

```suggestion
    "icon": "👨‍👩‍👧‍👦"
```